src/database.ts

```diff
@@ -2221,12 +2225,11 @@ class Database extends common.GrpcServiceObject {
             options.requestOptions
           );
         }
-        if (options.optimisticLock) {
-          transaction!.useOptimisticLock();
-        }
-        if (options.excludeTxnFromChangeStreams) {
-          transaction!.excludeTxnFromChangeStreams();
-        }
+        transaction?.setReadWriteTransactionOptions(
+          options && Object.keys(options).length
```

Comment:
How would this handle the case, when `TransactionOptions` only has `excludeTxnFromChangeStreams` set and `isolation_level` needs to be taken from `defaultTransactionOptions`